### DISCUSSION TITLE
Tokenomics

### DISCUSSION THREAD

**Discussion by hewison-chris** [bosagora/AgoraMainNet#30] [General]
Are the current values in the Bosagora Whitepaper for tokenomics sensible?

This shows the number of validators possible and the return on investment depending what percentage is staked:

rewards year 1 | 170,294,400 |   |  
-- | -- | -- | --
  |   |   |  
Initial supply | 450,000,000 |   |  
  |   |   |  
Stake per validator | 40,000.00 |   |  
  |   |   |  
% staked | staked | Number of validators | APR
100% | 450000000 | 11,250 | 37.8432
75% | 337500000 | 8,438 | 50.4576
50% | 225000000 | 5,625 | 75.6864
25% | 112500000 | 2,813 | 151.3728
10% | 45000000 | 1,125 | 378.432
5% | 22500000 | 563 | 756.864
1% | 4500000 | 113 | 3784.32

What is the desired % staked and how many validators do we need to be secure?

**Comment by omerfirmak** [comment]
So, we went with ethereum PoS because it supports hundreds of thousands of validators even though our validator set size was capped at 11k.


I feel like we might need to decrease the stake amount, so that we will likely never see the insane APRs that we get with a small set of validators.

**Comment by hewison-chris** [comment]
Even if the stake is reduced the APR will be the same as it is
```
yearly rewards * 100 / total staked. 
```
Just the number of validators possible increases.

I think we need to reduce the reward allocation to the validators.

**Comment by omerfirmak** [comment]
Yes, but it will lower the barrier to entry as well. So in the end we should have more BOAs staked.
